### ISSUE TITLE
[False-Positive] Makes no sense, because this type is not defined in annotations - Wrong return type for iconv

### ISSUE THREAD

**Issue by andreasschroth** [kalessil/phpinspectionsea#1070]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Issue type     | False-Positive                          |
| Plugin         | Php Inspections (EA Ultimate), 2.0.8 |
| Language level | PHP 7.2                        |

Current behaviour:
---

Here is some example code:

```php
$foo = $_GET['foo'] ?? '';
$foo = iconv('UTF-8', 'US-ASCII//TRANSLIT', $foo);
if ($foo === false) {
    throw new \RuntimeException('Could not transform foo parameter.');
}
```

I get displayed a warning from the "Makes no sense, because this type is not defined in annotations" inspection: The plugin thinks `iconv()` can only return a string, which is not true. In fact, on errors, it returns `false`.

Expected behaviour:
---

Do not warn in this case, as the check is useful.

**Comment by kalessil** [comment]
Checked: the function signature in PhpStorm stubs is incorrect. BTW, you can contribute a fix there.

**Comment by andreasschroth** [comment]
Thank you, PR submitted over there.
